### ISSUE TITLE
wheel not found, prevents pip install on Python 3.7.2, parasail-python 1.1.16

### ISSUE THREAD

**Issue by daveuu** [jeffdaily/parasail-python#34]
I'm not sure if this is caused by a problem with my virtualenv or parasail-python packaging, but I'll post this in case of the latter:

```
$ python3 --version
Python 3.7.2
$ python3 -m venv ~/venv_parasail
$ ~/venv_parasail/bin/pip install --upgrade pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/d8/f3/413bab4ff08e1fc4828dfc59996d721917df8e8583ea85385d51125dceff/pip-19.0.3-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 18.1
    Uninstalling pip-18.1:
      Successfully uninstalled pip-18.1
Successfully installed pip-19.0.3
$ ~/venv_parasail/bin/pip install --no-cache-dir parasail
Collecting parasail
  Downloading https://files.pythonhosted.org/packages/26/68/2e051763c912e9814c5bd8f4b0b9ca3e846cf62c44a3b250852298454482/parasail-1.1.16.tar.gz (70kB)
    100% |████████████████████████████████| 71kB 10.4MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-be4begcq/parasail/setup.py", line 18, in <module>
        from wheel.bdist_wheel import bdist_wheel as bdist_wheel_
    ModuleNotFoundError: No module named 'wheel'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-be4begcq/parasail/
```

So the pip install failed without the wheel package.

```
$ ~/venv_parasail/bin/pip install wheel
Collecting wheel
  Using cached https://files.pythonhosted.org/packages/96/ba/a4702cbb6a3a485239fbe9525443446203f00771af9ac000fa3ef2788201/wheel-0.33.1-py2.py3-none-any.whl
Installing collected packages: wheel
Successfully installed wheel-0.33.1
$ ~/venv_parasail/bin/pip install --no-cache-dir parasail
Collecting parasail
  Downloading https://files.pythonhosted.org/packages/26/68/2e051763c912e9814c5bd8f4b0b9ca3e846cf62c44a3b250852298454482/parasail-1.1.16.tar.gz (70kB)
    100% |████████████████████████████████| 71kB 9.9MB/s 
Collecting numpy (from parasail)
  Downloading https://files.pythonhosted.org/packages/61/57/07c49e1a6d2706fb7336b3fb11dd285c1e96535c80833d7524f002f57086/numpy-1.16.1-cp37-cp37m-manylinux1_x86_64.whl (17.3MB)
    100% |████████████████████████████████| 17.3MB 71.3MB/s 
Installing collected packages: numpy, parasail
  Running setup.py install for parasail ... -
```
etc.
So the work around is trivial but maybe wheel should be a parasail-python dependency? (or maybe it should always be in a virtualenv?)

Thanks for all your work on parasail.

**Comment by jeffdaily** [comment]
Fixed just now.  Thank you for the detailed report.  I will push a new pip package as soon as I can.

**Comment by blaiseli** [comment]
I still observe this issue as of today (17/11/2019). Should this be re-opened?

**Comment by lucacozzuto** [comment]
I can confirm the problem on 26/02/2020

**Comment by jeffdaily** [comment]
New release tagged v1.2.  Please try again once pypi packages are available for 1.2.
